addons/account/static/src/js/reconciliation/reconciliation_action.js

```diff
@@ -109,7 +112,39 @@ var StatementAction = AbstractAction.extend(ControlPanelMixin, {
 
         this.renderer.prependTo(self.$('.o_form_sheet'));
         this._renderLines();
-        this._openFirstLine();
+
+        // No more lines to reconcile, trigger the rainbowman.
+        var initialState = this.renderer._initialState;
+        if(initialState.valuenow === initialState.valuemax){
```

Comment:
if we perform the change explained in my other comments this can be rewritten as easily as: 
if (no_line_to_reconcile and presence_of_reconcile_aml_ids) {
    showRainbowMan
}
else {
    openFirstLine()
}